### PR TITLE
We've already fetched the extras, no need to fetch them again

### PR DESCRIPTION
**Pull request by StijnVrolijk**
## Type
- Enhancement

## Pull request description
We fetched the extras twice in the page edit action. Now we only do it once, +1 performance points!

### CODE REVIEW

**Comment by jeroendesloovere** [comment]
Nice catch

File: src/Backend/Modules/Pages/Actions/Edit.php [hunk 1]
```diff
@@ -635,7 +635,7 @@ protected function parse(): void
         $this->template->assign('templates', $this->templates);
         $this->template->assign('positions', $this->positions);
         $this->template->assign('extrasData', json_encode(BackendExtensionsModel::getExtrasData()));
-        $this->template->assign('extrasById', json_encode(BackendExtensionsModel::getExtras()));
+        $this->template->assign('extrasById', json_encode($this->extras));
```
Comment (WouterSioen) [inline src/Backend/Modules/Pages/Actions/Edit.php]: The question is, can these extras be changed in between fetching them and assigning them here? 🤔

Comment (StijnVrolijk) [reply src/Backend/Modules/Pages/Actions/Edit.php]: well.. they can but they aren't. They're used to check if a page contains more than 1 block or if a block is placed on the homepage. Then they get assigned.

Comment (WouterSioen) [reply src/Backend/Modules/Pages/Actions/Edit.php]: Let's 🚢 then!

**Pull request merged**